package.json

```diff
@@ -14,15 +14,16 @@
   "main": "index.js",
   "scripts": {
     "test": "lab -cv",
-    "lint": "jshint test index.js --reporter node_modules/jshint-stylish/stylish.js --exclude node_modules"
+    "lint": "jshint test index.js --reporter node_modules/jshint-stylish/index.js --exclude node_modules"
```

Comment:
@erikkemperman yeah, we've been discussing it a lot on a few PRs and that was the outcome.  My personal style is quite different from the gulp style, so it will probably require a bunch of changes.